lms/djangoapps/courseware/views/views.py

```diff
@@ -267,12 +268,16 @@ def courses(request):
     # Add marketable programs to the context.
     programs_list = get_programs_with_type(request.site, include_hidden=False)
 
+    # Get upcoming webinars to display them on `learn` page
+    upcoming_webinars = Webinar.objects.filter(status=Webinar.UPCOMING).order_by('start_time')
```

Comment:
Shouldn't it be descending?

Reply:
No, we need to display them in [ascending order](https://philanthropyu.atlassian.net/browse/LP-2623)